IrConsoleApp/Lecture4/Lecture4Homework.cs

```diff
@@ -0,0 +1,137 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace IrynaZharnasek.Lecture4
+{
+    public class Lecture4Homework
+    {
+        public static void Task1()
+        {
+            Console.WriteLine("Enter any positive number:");
+            int pNumb = Convert.ToInt32(Console.ReadLine());
+            int sum;
+            for (int i = 0; i < pNumb; i++)
+            {
+                sum = i + (i + 1);
+                Console.WriteLine($"The sum of numbers up to {pNumb} is:{sum}");
+            }
+        }
+
+        public static void Task2()
```

Comment:
The goal of the task is reached, it works, but you used do... while.. loop instead of while. Please check how it can be done with while loop

Reply:
Done